docs/README.md

```diff
@@ -83,41 +83,47 @@ The Notebooks can run on various compute targets; the ones currently supported o
+
+Generally, predictions can be made either in real time or on a batch basis. This solution was built to support near real-time featurization and scoring, which means up-to-date predictions are generated almost as soon as new data is available.
 
 ## Model operationalization
 
+When using Azure Machine Learning (currenty *Internal Preview)*, it takes the following steps to operationalize a model:
 
+* Registration
+* Docker image creation
+* Deployment of the image to a compute target
 
-## Featurization
-![](img/productionalization_feature_engineering.png)
-## Scoring, visualization and actions
-![](img/productionalization_scoring.png)
+The end result is a real-time scoring Web service with a REST API interface.
 
-# Using the Dashboard
+The solution includes a pre-trained model, which is deployed to Azure Container Instances (ACI) and used as part of the pre-configured data pipeline to score all new feature data. It is possible (and, in fact, recommended) to replace this default model with a custom one by following the modeling steps.
 
-# Further reading
+## Telemetry ingestion and featurization
 
-<div class="github-only">
-## Data Ingress and Storage
+The solution employs [Spark Structured Streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html) to perform the featurization of IoT telemetry. The streaming job runs on Azure Databricks.
 
-Solving any business problem with AI starts with data. The question you are trying to answer will affect what data you need, and in what format, quantities, and time horizons.  Most predictive maintenance problems are based on the desire to understand the behavior and operational health of distributed devices with embedded sensors. A message ingestion service supports ingesting real-time operational data from those sensors and saving it to long term (cold) cloud storage.  For most predictive maintenance scenarios, this message ingestion substrate also provides two-way communication back to remote devices to configure and manage those devices over time.
+Briefly, [Azure Event Hubs Connector for Apache Spark](https://github.com/Azure/azure-event-hubs-spark) allows making the *Event Hub-compatible endpoint* of the *IoT Hub* a streaming input source. The streaming job (written in Scala) extracts features from the incoming telemetry augmented with historical and static data persisted in a data store (e.g., Azure Table Storage). This exactly mimics the feature engineering process performed during modeling. The main difference is that here it happens in near real time. Notice the feedback loop in the diagram below: its purpose is the augmentation of new data with the results of previous aggregations, in other words, the historical data.
```

Comment:
[Azure Event Hubs Connector for Apache Spark](https://github.com/Azure/azure-event-hubs-spark) enables connecting  *IoT Hub* to a streaming input source via an *Event Hub-compatible endpoint*. The streaming job extracts features from the incoming telemetry augmented with historical and static data persisted in a data store (e.g., Azure Table Storage). This exactly mimics the feature engineering process performed during modeling. The main difference is that here it happens in real-time. Notice the feedback loop in the diagram below: its purpose is the augmentation of new data with the results of previous aggregations, in other words, the historical data.